definitions/npm/ramda_v0.26.x/flow_v0.104.x-/test_ramda_v0.26.x_relation.js

```diff
@@ -314,3 +313,19 @@ const sym: Array<number> = _.symmetricDifference([1, 2, 3, 4], [7, 6, 5, 4, 3]);
 
 const un: Array<number> = _.union([1, 2, 3])([2, 3, 4]);
 const un1: Array<{ [k: string]: number, ... }> = _.unionWith(eqA, ls1, ls2);
+
+// Given $ReadOnlyArray is super type of Array, no need to test Array -> $ReadOnlyArray case
+describe('intersection', () => {
+  const readOnlyArray: $ReadOnlyArray<number> = [1, 2, 3]
+  it('should only accept read only input', () => {
+    const inters1: $ReadOnlyArray<number> = intersection(readOnlyArray, readOnlyArray)
+    const inters2: $ReadOnlyArray<number> = intersection(readOnlyArray)(readOnlyArray)
+  });
+
+  it('should error out when output is not read only', () => {
```

Comment:
This test will need to go away, but we should still test that we can freely mix `Array` and `$ReadOnlyArray`.

Reply:
Done